prime-router/docs/api/check.yml

```diff
@@ -17,7 +17,61 @@ paths:
         '200':
           description: success call
 
+  /api/checkreceiver/org/{orgName}/setting/{settingName}:
+    post:
+      description: Load Receiver Settings and verifies connecting to the external server works.
+      security:
+        - OAuth2: [ org_admin, system_admin ]
+      parameters:
+        - name: orgName
+          in: path
+          description: Name of Organization.
+          required: true
+          schema:
+            type: string
+        - name: settingName
+          in: path
+          description: Logical name of the Receiver Setting
+          required: true
+          schema:
+            type: string
+      responses:
+        '200':
+          description: The table contents
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/CheckReceiverResponse'
+        '404':
+          description: The orgName or settingName  was not found
+          content:
+            text/plain:
+              schema:
+                type: string
+
 components:
   securitySchemes:
     OAuth2:
       $ref: 'https://raw.githubusercontent.com/CDCgov/prime-reportstream/master/prime-router/docs/api/components/security_schemes.yml#/OAuth2'
+
+  schemas:
+    ErrorMsg:
```

Comment:
schema ErrorMsg: is syntactically correct, but is unused.

Reply:
Ah good catch. This is what's returned when there's a 4xx, 5xx response.

Reply:
Fixed by removing. Did add an example of error message text